plugin-template

```diff
@@ -15,6 +15,7 @@ from jinja2 import Environment, FileSystemLoader
 
 
 DEFAULT_SETTINGS = {
+    'additional_plugins': [],
```

Comment:
@dkliban what do you think about making `pulp-certguard` default?
`'additional_plugins': ['pulp-certguard'],`

Reply:
Let's leave it empty for now. However, when we update the template_config.yml for pulpcore, we definitely want to add 'pulp-certguard' to hte list there.